tests/integration/test_storage_delta/test.py

```diff
@@ -387,7 +506,7 @@ def test_metadata(started_cluster, use_delta_kernel):
     assert next(iter(stats["minValues"].values())) == 0
     assert next(iter(stats["maxValues"].values())) == 99
 
-    create_delta_table(instance, TABLE_NAME, use_delta_kernel=use_delta_kernel)
+    instance.query(create_delta_table("s3", TABLE_NAME, started_cluster, use_delta_kernel = use_delta_kernel))
```

Comment:
Why not do `instance.query` inside `create_delta_table` as it was before?

Reply:
addressed it